source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,845 @@
--------------------------
+Editing an existing local bundle is as simple as opening the bundle definition
+file in your favorite editor, making the desired edits, and saving your changes.
 
-If you initialized your workspace to be tracked as a git repository
-with the :command:`mixer init --git` command, it might be useful to apply a
-git commit after you modify the mix bundle list or edit a bundle definition
-file.
+A note on removing bundles from a mix: By default, removing a bundle will only
+remove the bundle from the mix. The local bundle defintion file will still
+remain. To completely remove a bundle, including its local bundle definition
+file, use the :command:`--local` flag.
 
-All the :command:`mixer bundle` commands in the previous sections support an
-optional *--git* flag. This flag automatically applies a git commit
-when the command completes, for example:
+If you remove the bundle definition file for a local, edited version of an
+upstream bundle in a mix, the mix will revert to reference the original upstream
+version of the bundle.
 
-.. code-block:: bash
+Configure and enable Docker
+===========================
 
-   mixer bundle remove --git bundle1
+Use these steps to enable Docker for the mixer tool. Make sure to
+`Configure Docker proxy info`_ first if needed.
 
-Build the bundle chroots
-========================
+#. Start the Docker daemon:
 
-To build all the ``chroots`` based on the defined bundles, use the following
-command in your workspace:
+   .. code-block:: bash
 
-.. code-block:: bash
+      sudo systemctl start docker
+      sudo chmod 777 /var/run/docker.sock
+      sudo docker info
 
-   mixer build bundles 
+#. Add user to the docker group
 
-If the mix has many bundles, this step might take some time.
+   .. code-block:: bash
 
-Mixer automatically gathers the bundle definition files for the upstream
-bundles into a :file:`upstream-bundles` directory, and user bundles should
-be placed directly into :file:`local-bundles`. The local path is set in
-the `LOCAL_BUNDLE_DIR` variable in the :file:`builder.conf`. **Do not edit
-files in upstream-bundles.** Mixer automatically deletes the contents of
-the :file:`upstream-bundles` directory before repopulating the directory
-on-the-fly if a new version must be downloaded.
+      sudo usermod -G docker -a <username>
 
+The mixer Docker container is available on `Docker Hub`_. mixer will
+automatically pull a Docker container for mixing if one does not already exist.
+You can optionally pull a container in advance, using the following steps:
 
-Create an update
-================
+#. Find the latest version of the container by viewing the tags for the
+   `clearlinux/mixer repo <https://hub.docker.com/r/clearlinux/mixer/tags/>`_
+   on Docker Hub.
 
-Create an update with the following command:
+#. Pull the latest container version:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   mixer build update
+      docker pull clearlinux/mixer:<version>
```

Comment:
`<version>` is actually the upstream format.

Reply:
@rchiossi The previous step instructs to "Find the latest version of the container by viewing the tags for the clearlinux/mixer repo on Docker Hub." In this step then, should it be clarified that <version> is actually the latest container version (from the previous step)? e.g. docker pull clearlinux/mixer:<docker-container-version>

Reply:
It is not the the docker container version, it is the upstream format. 
`docker pull clearlinux/mixer:<upstream-format>`  is a more accurate description.

Reply:
Just to clarify, for a given upstream format there can be multiple docker container versions, although only the latest version for a given format will be visible in dockerhub.